### DISCUSSION TITLE
Derive multiple wallet addresses from the same authentication in web3auth

### DISCUSSION THREAD

**Discussion by jasalo** [Web3Auth/Web3Auth#613] [Help]
How can you derive multiple wallet addresses from the same authentication in web3auth?

**Comment by himanshuchawla009** [comment]
You can use private key returned by web3auth as a seed entropy to create a seed phrase and use that seed phrase in bip44 to generate hierarchically deterministic accounts.

**Comment by yashovardhan** [comment]
Duplicate of [#438](https://github.com/orgs/Web3Auth/discussions/438)
